CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Improved app performance on first login by requesting fewer events from relays.
```

Comment:
I know this is the right place _right now_ but since we're in the middle of a release I want to flag it. If this gets merged before we stamp the release, the CHANGELOG will say this is in 0.1.23 when, in fact, it's not.

Reply:
Oops, I just realized that we run the stamp_release on the release branch, so this should be no big deal.